### PR TITLE
Add links to cards.

### PR DESCRIPTION
**Pull request by minecraftchest1**
**What kind of change does this PR introduce?**

Half of the cards on the site now link to more info.

**What is the current behavior?**

Cards just show information.

**What is the new behavior?**

Cards are now links as well.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/AvaloniaUI/avaloniaui.net?pullRequest=261) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/AvaloniaUI/avaloniaui.net?pullRequest=261) before we can accept your contribution.<br/><br/>:x: minecraftchest1 [sign now](https://cla.dotnetfoundation.org/AvaloniaUI/avaloniaui.net?pullRequest=261)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/AvaloniaUI/avaloniaui.net?pullRequest=261) it.</sub>

**Comment by minecraftchest1** [comment]
Except for the mobile support card, this PR is complete.

**Comment by minecraftchest1** [comment]
Any Update?

**Comment by robloo** [comment]
There have been several fixes for spelling/grammar that this PR will conflict with. Updates would need to be made to it now (assuming the PR is directionally correct).

**Comment by Arhell** [comment]
сс @minecraftchest1 resolve conflicts please

**Comment by minecraftchest1** [comment]
That should be fixed.

File: index.html [hunk 1]
```diff
@@ -276,25 +284,29 @@ <h6 class="card-title">Cross Platform</h6>
...
+						</div>
+					</div>
+				</a>
+				
+				<a class="linkcard" href="support.html">
+					<div class="card mt-5">
+						<div class="card-body d-flex flex-column">
+							<img class="img-fluid mt-3 ms-3" src="assets/Help.svg" width="92" height="92" />
+							<div class="mt-auto ms-3 mb-3">
+								<h6 class="card-title">Commerical Support</h6>
```
Comment (robloo) [inline index.html:304]: You still bring back spelling errors that were fixed.

File: index.html [hunk 2]
```diff
@@ -211,37 +211,45 @@ <h4>We have an uncompromising focus on creating a cross-platform UI framework th
...
+					
+						<div class="card-body d-flex flex-column">
+							<img class="img-fluid mt-3 ms-3" src="assets/jb-colour.svg" width="92" height="92" />
+							<div class="mt-auto ms-3 mb-3">
+								<h6 class="card-title">JetBrains Rider & R#</h6>
+								<p class="card-text">JetBrains deep understanding of Avalonia shines through with rich Rider and R# support.</p>
+							</div>
+						</div>
+					</div>
+        </a>
```
Comment (robloo) [inline index.html:249]: Wrong indentation

File: index.html [hunk 3]
```diff
@@ -211,37 +211,45 @@ <h4>We have an uncompromising focus on creating a cross-platform UI framework th
...
+					
+						<div class="card-body d-flex flex-column">
+							<img class="img-fluid mt-3 ms-3" src="assets/VS.svg" width="92" height="92" />
+							<div class="mt-auto ms-3 mb-3">
+								<h6 class="card-title">Visual Studio Extension</h6>
+								<p class="card-text">The free VS extension provides everything needed to start, including a XAML previewer.</p>
+							</div>
+						</div>
+					</div>
+                </a>
```
Comment (robloo) [inline index.html:237]: nit: Indentation again

File: index.html [hunk 4]
```diff
@@ -276,25 +284,29 @@ <h6 class="card-title">Cross Platform</h6>
...
+					<div class="card mt-5">
+						<div class="card-body d-flex flex-column">
+							<img class="img-fluid mt-3 ms-3" src="assets/Help.svg" width="92" height="92" />
+							<div class="mt-auto ms-3 mb-3">
+								<h6 class="card-title">Commerical Support</h6>
+								<p class="card-text">Get the support your project needs with one of our commercial support agreements. Help you can count on.</p>
+							</div>
+						</div>
+					</div>
+                </a>
```
Comment (robloo) [inline index.html:309]: Indentation of all </a> tags should be re-reviewed. This HTML was formatted nicely and care should be taken to ensure that isn't lost.

**Comment by minecraftchest1** [comment]
Edits by maintainers is allowed.

File: index.html (hunk 1 above)
Comment (minecraftchest1) [reply index.html:304]: I must've copied the wrong text. I'll fix it when I get a chance.

File: index.html (hunk 3 above)
Comment (minecraftchest1) [reply index.html:237]: How is it indented differently? How do you want it indented?

File: index.html (hunk 2 above)
Comment (minecraftchest1) [reply index.html:249]: How is it indented differently? How do you want it indented?

**Comment by minecraftchest1** [comment]
I created a new PR at #287 rather then trying how to update this one.

**Review by Arhell — COMMENTED** [review]
Thanks for your contribution

**Comment by Arhell** [comment]
> I created a new PR at #287 rather then trying how to update this one.



> I created a new PR at #287 rather then trying how to update this one.

close

**Pull request closed without merging**